tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/AuthorisedSteps.cs

```diff
@@ -66,28 +64,29 @@ public async Task GivenAnAuthorityUserIsLoggedInWithUsernameAndPassword(Table ta
                 return;
             }
 
-            _context[_contextConstants.AccessTokenKey] = tokenResponse.AccessToken;
+            _context[ScenarioContextKeys.AccessTokenKey] = tokenResponse.AccessToken;
         }
 
         [Given(@"the claims contains the following information")]
         public void GivenTheClaimsContainsOrganisation(Table table)
         {
-            var expectedClaims = table.Rows.First().Select(x => (x.Key, x.Value));
+            var expectedClaims = table.CreateSet<(string, string)>();
```

Comment:
I thought we created a Claims table here?

Reply:
We have evolved beyond claims tables

Reply:
^What @CallumBradburyBJSS  said

Reply:
I'm not sure it is the right approach you lose the context of what each string string means, now I have to go to the feature file or look at the rest of the code to figure out what string string refers to. To mitigate this are you able to label the tuple properties?

Reply:
You sure can, I sent Ruth a code snippet of how to do it, as I also wasn't happy with the lack of context